tests/files/mod.rs

```diff
@@ -1,3 +1,31 @@
 // Copyright (c) ZeroC, Inc.
 
 mod io;
+
+use slicec::diagnostics::Diagnostics;
+use slicec::slice_file::SliceFileHashable;
+use slicec::slice_options::SliceOptions;
+use slicec::utils::file_util::resolve_files_from;
+use std::path::PathBuf;
+
+/// This test is used to verify that the `hash_all` method returns a hash that is stable across releases and uses
+/// the correct ordering of files.
```

Comment:
It would be worth testing the reverse order is also the same. No need to hardcode the hash. Just test the hash is the same in both orders.
```suggestion
///  Test that `compute_sha256_hash` returns a hash that is order independent.
```